manifests/v1beta1/components/controller/controller.yaml

```diff
@@ -18,7 +18,6 @@ spec:
       annotations:
         prometheus.io/scrape: "true"
         prometheus.io/port: "8080"
-        sidecar.istio.io/inject: "false"
```

Comment:
We need this flag added for standalone mode

Reply:
I've updated the PR!